service/src/main/java/uk/gov/hmcts/reform/fpl/config/DefaultEmailLookupConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class DefaultEmailLookupConfiguration {
```

Comment:
I do not see much value for that class. It could be simplified by injecting value directly via `@Value("${fpl.default_email.mapping}")` to `InboxLookupService`.

Reply:
Done this change.